doc/managing-certificates-for-application-ingress.md

```diff
@@ -0,0 +1,34 @@
+# Managing certificates for application ingress
+
+For each domain in CF:
+
+0. [Make sure SDS is enabled for Istio](https://istio.io/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-tls-ingress-gateway-using-sds)
+
+1. Create a secret with the certificate and the key in `istio-system` namespace:
+```
+kubectl create secret tls wildcard-apps-example-com-cert \
+    -n istio-system \
+    --cert='path/to/wildcard-apps.example.com.crt' \
+    --key='path/to/wildcard-apps.example.com.key'
+```
+
+_Note: This can be either a wildcard cert or a cert for a single domain._
+
+2. Add new server in the edge Istio gateway object in cf-workloads, e.g.
+```
+- hosts:
+  # the host from your certificate
+  - '*.apps.example.com'
+  port:
+    name: https-apps-example-com
+    number: 443
+    protocol: HTTPS
+  tls:
+    # secret created in first step
+    credentialName: wildcard-apps-example-com-cert
+    mode: SIMPLE
+```
```

Comment:
What is the command to create a new server?

Reply:
@Syerram

I'm not sure I understand. I did update the wording for this step since we found it confusing, but this is just an entry in an array of `spec.servers` on the `Gateway`.

Reply:
@tcdowney Thx. I didn't know how an operator would add the above config. From the updated doc, the operator will execute `kubectl edit <gateway resource>` (or k8s patch) to add the above config to the Gateway resource.